### ISSUE TITLE
About the YMF825 protocol

### ISSUE THREAD

**Issue by denjhang** [danielrfry/opl2sd1#18]
There are already some software that can directly control the YMF825 and can make full use of the new features of the YMF825:
1. YMF825 tone editor written in Java:https://github.com/shigosenCCneko/YMF825MIDI_CONTROL_for_Arduino
2. YMF825 tone editor written in C#:https://github.com/nanase/ymf825

**Comment by danielrfry** [comment]
Indeed, there is also my own MIDI implementation project, [Note Salad](https://github.com/danielrfry/notesalad), that can adjust YMF825 FM parameters via NRPN messages 😄
https://youtu.be/qVMLqGQ5HB0

`opl2sd1` though is specifically about providing an OPL compatible interface, so adding direct control of the YMF825 is out of scope I think.
